tests/e2e/test_token_details.py

```diff
@@ -6,24 +6,19 @@
 
 
 class TestTokenDecimals(unittest.TestCase):
-    def setUp(self) -> None:
-        self.cow = "0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB"
-        self.usdc = "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"
-
     def test_token_decimals(self):
-        self.assertEqual(get_token_decimals(self.cow), 18)
-        self.assertEqual(get_token_decimals(self.usdc), 6)
-
-        self.assertEqual(get_token_decimals(duneapi.types.Address(self.usdc)), 6)
-        self.assertEqual(get_token_decimals(duneapi.types.Address(self.cow)), 18)
+        # Goerli doesn't seem to have tokens with anything other than 18 decimals.
+        cow = "0x3430d04E42a722c5Ae52C5Bffbf1F230C2677600"
+        self.assertEqual(get_token_decimals(cow), 18)
+        self.assertEqual(get_token_decimals(duneapi.types.Address(cow)), 18)
 
     def test_token_decimals_cache(self):
-        new_token = "0x10245515d35BC525e3C0977412322BFF32382EF1"
+        usdc = "0x5ffbac75efc9547fbc822166fed19b05cd5890bb"
```

Comment:
Nitpick: In my opinion all constants should be all uppercase. What's your take on this?

Ref: https://peps.python.org/pep-0008/#constants

Reply:
Constants are only uppercase as GLOBAL variables (outside of method declarations). inside of a method they are not consider constants.

Reply:
Unless we want to explicitly disagree with PyCharm's conventions.

<img width="611" alt="Screenshot 2022-10-18 at 12 20 55 PM" src="https://user-images.githubusercontent.com/11778116/196404397-dede4965-835f-45f6-9e93-077a12824b6a.png">

Reply:
Ah I didn't know that. Thanks for sharing this with me.

Please proceed without implementing the suggested changes then.